nasa/earthdata-search
# EDSC-3475: Improves SEO meta information

# Overview

### What is the feature?

Improves SEO meta information within Earthdata Search

### What is the Solution?

Added Helmet components to the relevant routes to add SEO meta tags.

### What areas of the application does this impact?

 - Search Results
 - Granule Results
 - Collection Details
 - Collection Subscriptions
 - Granule Subscriptions
 - Download History
 - Order Status
 - Preferences
 - Contact Info
 - Saved Projects
 - Project

For each affected area, the following should be set:
 - description
 - title
 - thumbnail
 - robots
 - canonical link

Additionally, only one h1 tag should exist on each page

# Testing

### Reproduction steps

- **Environment for testing:** Any
- **Collection to test with:** Any

For each affected page,
1. Visit the page
2. Confirm the meta information looks correct using "Meta SEO Inspector" extension

# Checklist
